web/rootfs/defaults/settings-config.js

```diff
@@ -22,6 +22,12 @@
 {{ $ENABLE_TALK_WHILE_MUTED := .Env.ENABLE_TALK_WHILE_MUTED | default "false" | toBool -}}
 {{ $ENABLE_TCC := .Env.ENABLE_TCC | default "true" | toBool -}}
 {{ $ENABLE_TRANSCRIPTIONS := .Env.ENABLE_TRANSCRIPTIONS | default "false" | toBool -}}
+{{ $TRANSLATION_LANGUAGES := .Env.TRANSLATION_LANGUAGES -}}
```

Comment:
Doesn't this need a default?

Reply:
Sorry, I'm confused, should I add a default? Your previous change request said we shouldn't add defaults.

Would an empty default like below be appropriate?
`{{ $TRANSLATION_LANGUAGES := .Env.TRANSLATION_LANGUAGES | default "[]" -}}`

Reply:
Yeah, that looks ok.

Reply:
Alright, fixed 😄